src/Illuminate/Concurrency/composer.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "illuminate/concurrency",
+    "description": "The Illuminate Concurrency package.",
+    "license": "MIT",
+    "homepage": "https://laravel.com",
+    "support": {
+        "issues": "https://github.com/laravel/framework/issues",
+        "source": "https://github.com/laravel/framework"
+    },
+    "authors": [
+        {
+            "name": "Taylor Otwell",
+            "email": "taylor@laravel.com"
+        }
+    ],
+    "require": {
+        "php": "^8.0.2",
```

```suggestion
        "php": "^8.2",
```

Illuminate/Process v11 requires PHP 8.2:
https://github.com/illuminate/process/blob/69df5be9de8389a2aed90dc2555b898ee0f3d231/composer.json#L17